src/main/java/io/jenkins/plugins/opentelemetry/job/cause/BitBucketPushCauseHandler.java

```diff
@@ -12,17 +12,21 @@
 import javax.annotation.Nonnull;
 
 @Extension(optional = true, dynamicLoadable = YesNoMaybe.YES)
-public class BitBucketPushCauseHandler extends AbstractCauseHandler {
+public class BitBucketPushCauseHandler implements CauseHandler {
 
     @Override
     public boolean isSupported(@Nonnull Cause cause) {
         return isBitBucketPushCause(cause);
     }
 
+    protected boolean isBitBucketPushCause(Cause cause) {
+        return cause instanceof com.cloudbees.jenkins.plugins.BitBucketPushCause;
```

Comment:
```suggestion
        return cause.getClass().getName().equals("com.cloudbees.jenkins.plugins.BitBucketPushCause");
```

Reply:
Probably this can be merged into `isSupported`

Reply:
Fixed slightly differently with https://github.com/jenkinsci/opentelemetry-plugin/pull/208/commits/f3a999c7f9a2a1929185d27c7bf2f7f44486b388